<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.io.smart.web.rest.errors.ExceptionTranslatorIntTest" tests="11" skipped="0" failures="0" errors="0" timestamp="2018-02-10T05:31:56" hostname="Nicholass-MacBook-Air.local" time="2.646">
  <properties/>
  <testcase name="testMissingServletRequestPartException" classname="com.io.smart.web.rest.errors.ExceptionTranslatorIntTest" time="0.582"/>
  <testcase name="testMissingServletRequestParameterException" classname="com.io.smart.web.rest.errors.ExceptionTranslatorIntTest" time="0.203"/>
  <testcase name="testAccessDenied" classname="com.io.smart.web.rest.errors.ExceptionTranslatorIntTest" time="0.181"/>
  <testcase name="testParameterizedError" classname="com.io.smart.web.rest.errors.ExceptionTranslatorIntTest" time="0.205"/>
  <testcase name="testInternalServerError" classname="com.io.smart.web.rest.errors.ExceptionTranslatorIntTest" time="0.208"/>
  <testcase name="testMethodNotSupported" classname="com.io.smart.web.rest.errors.ExceptionTranslatorIntTest" time="0.175"/>
  <testcase name="testExceptionWithResponseStatus" classname="com.io.smart.web.rest.errors.ExceptionTranslatorIntTest" time="0.161"/>
  <testcase name="testMethodArgumentNotValid" classname="com.io.smart.web.rest.errors.ExceptionTranslatorIntTest" time="0.366"/>
  <testcase name="testParameterizedError2" classname="com.io.smart.web.rest.errors.ExceptionTranslatorIntTest" time="0.173"/>
  <testcase name="testUnauthorized" classname="com.io.smart.web.rest.errors.ExceptionTranslatorIntTest" time="0.199"/>
  <testcase name="testConcurrencyFailure" classname="com.io.smart.web.rest.errors.ExceptionTranslatorIntTest" time="0.172"/>
  <system-out><![CDATA[
        ██╗ ██╗   ██╗ ████████╗ ███████╗   ██████╗ ████████╗ ████████╗ ███████╗
        ██║ ██║   ██║ ╚══██╔══╝ ██╔═══██╗ ██╔════╝ ╚══██╔══╝ ██╔═════╝ ██╔═══██╗
        ██║ ████████║    ██║    ███████╔╝ ╚█████╗     ██║    ██████╗   ███████╔╝
  ██╗   ██║ ██╔═══██║    ██║    ██╔════╝   ╚═══██╗    ██║    ██╔═══╝   ██╔══██║
  ╚██████╔╝ ██║   ██║ ████████╗ ██║       ██████╔╝    ██║    ████████╗ ██║  ╚██╗
   ╚═════╝  ╚═╝   ╚═╝ ╚═══════╝ ╚═╝       ╚═════╝     ╚═╝    ╚═══════╝ ╚═╝   ╚═╝

:: JHipster ??  :: Running Spring Boot 1.5.9.RELEASE ::
:: http://www.jhipster.tech ::

2018-02-10 00:31:32.507  INFO 35660 --- [    Test worker] c.i.s.w.r.e.ExceptionTranslatorIntTest   : No active profile set, falling back to default profiles: default
2018-02-10 00:31:38.752  INFO 35660 --- [    Test worker] c.io.smart.config.MetricsConfiguration   : Initializing Metrics Log reporting
2018-02-10 00:31:46.712  INFO 35660 --- [    Test worker] c.io.smart.config.LoggingConfiguration   : Initializing Logstash logging
2018-02-10 00:31:46.895  INFO 35660 --- [    Test worker] c.io.smart.config.LoggingConfiguration   : Filtering metrics logs from all appenders except the LOGSTASH appender
2018-02-10 00:31:51.638  WARN 35660 --- [    Test worker] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2018-02-10 00:31:51.658  WARN 35660 --- [    Test worker] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2018-02-10 00:31:56.821  INFO 35660 --- [    Test worker] c.i.s.w.r.e.ExceptionTranslatorIntTest   : Started ExceptionTranslatorIntTest in 28.981 seconds (JVM running for 35.606)
2018-02-10 00:31:57.066  WARN 35660 --- [    Test worker] o.z.p.spring.web.advice.AdviceTrait      : Bad Request: Required request part 'missing Servlet request part' is not present
2018-02-10 00:31:57.634  WARN 35660 --- [    Test worker] o.z.p.spring.web.advice.AdviceTrait      : Bad Request: Required parameter type parameter 'missing Servlet request parameter' is not present
2018-02-10 00:31:57.817  WARN 35660 --- [    Test worker] o.z.p.spring.web.advice.AdviceTrait      : Forbidden: test access denied!
2018-02-10 00:31:58.005  WARN 35660 --- [    Test worker] o.z.p.spring.web.advice.AdviceTrait      : Bad Request: Parameterized Exception
2018-02-10 00:31:58.213 ERROR 35660 --- [    Test worker] o.z.p.spring.web.advice.AdviceTrait      : Internal Server Error

java.lang.RuntimeException: null
	at com.io.smart.web.rest.errors.ExceptionTranslatorTestController.internalServerError(ExceptionTranslatorTestController.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:205)
	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:133)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:97)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:827)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:738)
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:85)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:967)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:901)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:970)
	at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:861)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:687)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:846)
	at org.springframework.test.web.servlet.TestDispatcherServlet.service(TestDispatcherServlet.java:65)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at org.springframework.mock.web.MockFilterChain$ServletFilterProxy.doFilter(MockFilterChain.java:160)
	at org.springframework.mock.web.MockFilterChain.doFilter(MockFilterChain.java:127)
	at org.springframework.test.web.servlet.MockMvc.perform(MockMvc.java:155)
	at com.io.smart.web.rest.errors.ExceptionTranslatorIntTest.testInternalServerError(ExceptionTranslatorIntTest.java:144)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:75)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:86)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:84)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:252)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:94)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:191)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:745)

2018-02-10 00:31:58.409  WARN 35660 --- [    Test worker] o.z.p.spring.web.advice.AdviceTrait      : Method Not Allowed: Request method 'POST' not supported
2018-02-10 00:31:58.574  WARN 35660 --- [    Test worker] o.z.p.spring.web.advice.AdviceTrait      : Bad Request: null
2018-02-10 00:31:58.934  WARN 35660 --- [    Test worker] o.z.p.spring.web.advice.AdviceTrait      : Bad Request: Validation failed for argument at index 0 in method: public void com.io.smart.web.rest.errors.ExceptionTranslatorTestController.methodArgument(com.io.smart.web.rest.errors.ExceptionTranslatorTestController$TestDTO), with 1 error(s): [Field error in object 'testDTO' on field 'test': rejected value [null]; codes [NotNull.testDTO.test,NotNull.test,NotNull.java.lang.String,NotNull]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [testDTO.test,test]; arguments []; default message [test]]; default message [may not be null]] 
2018-02-10 00:31:59.119  WARN 35660 --- [    Test worker] o.z.p.spring.web.advice.AdviceTrait      : Bad Request: Parameterized Exception
2018-02-10 00:31:59.313  WARN 35660 --- [    Test worker] o.z.p.spring.web.advice.AdviceTrait      : Unauthorized: test authentication failed!
2018-02-10 00:31:59.490  WARN 35660 --- [    Test worker] o.z.p.spring.web.advice.AdviceTrait      : Conflict: test concurrency failure
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
